### PR TITLE
List found

### PR DESCRIPTION
**Pull request by levBagryansky**
@Graur , please review. 
I implemented list.found.

### CODE REVIEW

File: src/test/eo/org/eolang/collections/list-tests.eo [hunk 1]
```diff
@@ -381,3 +381,24 @@
...
+  assert-that > @
+    lst.found 2
+    $.equal-to 1
+
+[] > is-found-first-index
+  list > lst
+    * "qwerty" "asdfgh" 3 "qwerty"
+  assert-that > @
+    lst.found "qwerty"
+    $.equal-to 0
```
Comment (Graur) [inline src/test/eo/org/eolang/collections/list-tests.eo:404]: @levBagryansky Can you add this test:
```
[] > is-found-3
  list > lst
    * -1 2 -1
  assert-that > @
    lst.found -1
    $.equal-to 0
```

**Review by Graur — CHANGES_REQUESTED** [review]

File: src/test/eo/org/eolang/collections/list-tests.eo (hunk 1 above)
Comment (levBagryansky) [reply src/test/eo/org/eolang/collections/list-tests.eo:418]: added this and another one

**Review by Graur — APPROVED** [review]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**